### PR TITLE
implemented setBadgeCount (for Mac)

### PR DESCRIPTION
**Pull request by ColinEberhardt**
Currently delegates straight through to Electron's app.setBadgeCount.

Interestingly SymphonyElectron provide a Windows implementation, looks like it is an overlay on top of the app icon:

https://github.com/symphonyoss/SymphonyElectron/blob/master/js/badgeCount.js

### CODE REVIEW

File: packages/api-demo/src/app/app-api-demo.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+ssf.app.ready().then(() => {
+  document.getElementById('set-badge-count').onclick = () => {
+    ssf.app.setBadgeCount(Number(document.getElementById('badge-count').value));
+  };
+});
```
Comment (BenLambertNcl) [inline packages/api-demo/src/app/app-api-demo.js]: New-line

File: packages/api-electron/src/preload/app.ts [hunk 2]
```diff
@@ -1,5 +1,14 @@
+const {
+  remote
+} = require('electron');
+const electronApp = remote.app;
```
Comment (BenLambertNcl) [inline packages/api-electron/src/preload/app.ts]: If we only need one import here, might be better just to have:
`const electronApp = require('electron').remote.app`

**Review by BenLambertNcl — APPROVED** [review]
Very minor comments, looks good!

**Comment by ColinEberhardt** [comment]
Thanks @BenLambertNcl - updated with your suggestions

**Review by BenLambertNcl — APPROVED** [review]

**Review by andy-lee-eng — APPROVED** [review]

**Pull request merged**